sdk/src/beta9/cli/main.py

```diff
@@ -25,6 +25,8 @@
 
 click.formatting.FORCED_WIDTH = shutil.get_terminal_size().columns
 
+os.environ["GRPC_VERBOSITY"] = os.getenv("GRPC_VERBOSITY") or "ERROR"
```

Comment:
This should stop prevent the gRPC library logs from being emitted between builds and when thread/process pools are being used.